### ISSUE TITLE
feature request - swap multiple overrides with same value at once 

### ISSUE THREAD

**Issue by creativechris08** [ahmedmigo/overrideit-sketchplugin#15]
yo ahmedmigo,

Would it be possiple to add a option to swap multiple override values at once? such as all symbol overrides that are having the same value to be swapped with another value symbol, preferably using shared library’s 😎an example would be an artboard, page or symbol with the same x5 shared colour overrides values being swapped for another colour value in the same shared library at once 😎😎😎😎👍🏻

**Comment by Pausansol** [comment]
Yes, this is a good improvement. We are close to get a solution for this request ;)

**Comment by ahmedmigo** [comment]
Oh man it is super cool Idea! I really like it, I will add it to my feature list

**Comment by creativechris08** [comment]
😘😘😘😘
